src/LogExpert/Controls/LogWindow/LogWindowsPublic.cs

```diff
@@ -579,6 +578,26 @@ public void GotoLine(int line)
             }
         }
 
+        public void TryToTruncate()
+        {
+            try
+            {
+                if (LockFinder.CheckIfFileIsLocked(Title))
+                {
+                    var name = LockFinder.FindLockedProcessName(Title);
+                    var status = string.Format("Truncate failed: file is locked by {0}", name);
+                    StatusLineText(status);
+                    return;
+                }
+
+                File.WriteAllText(Title, "");
+            }catch(Exception E)
+            {
+                StatusLineText("Unexpected issue truncating file");
+                throw E;

```

Comment:
Don't re-throw exception, add the exception to the logger

Reply:
Done